nislmigrate/utility/information_logging.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from nislmigrate.logs.migration_error import MigrationError
+
+
+def list_installed_services_and_raise_if_capture_or_restore_present(argument_handler):
```

Comment:
"and" is often a bad sign in method names. It means a single method is doing too much. Can you refactor to separate concerns?

Reply:
Good point. I was able to refactor it to make it more clear (I think) and move the call to exit() to the top level call which makes more sense.